tests/qm-sanity-test/check_qm_with_aib.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Create AIB manifest with the correct QM copr
+USE_QM_COPR="${PACKIT_COPR_PROJECT:-rhcontainerbot/qm}"
+sed -e "s,\${QM_COPR},${USE_QM_COPR},g" check_qm_with_aib.aib.yml > qm.aib.yml
+
+AIB="automotive-image-builder build --export qcow2 qm.aib.yml qm.qcow2"
+BUILDDIR=_build
```

Comment:
from here on, it would be best to use the auto-image-builder.sh instead of copy&paste

Reply:
But then I would need to download it from gitlab? We don't have an EPEL package for `sample-images`, do we?

Reply:
correct. but it's still better to use that instead of this complex cmdline.
but yeah, we though about moving it into a-i-b repo instead...

Reply:
So I considered cloning the `sample-images` repo and run `auto-image-builder.sh` from there, but I thought that using the script from upstream/main might be unstable, and would add git installtion dependency. On the other hand, the beauty of having a container image with everything needed inside is that you don't really need to set anything up, and it runs with one command in (almost) any environment with podman. So actually I like one complex cmdline more than cloning a whole repo just to get the arguments straight. WDYT?

Reply:
Maybe putting `auto-image-builder.sh` in the container image would make sense? This would somewhat simplify this one-liner

Reply:
Please refer this issue
#675 now, it is more clear, so there is sample manifest and additional support in code https://gitlab.com/CentOS/automotive/src/automotive-image-builder/-/commit/f28b7f240a9a33f2273c2d8a578b3ebd4aad6623

simplifeid manifest format

Reply:
> which container do you want to include it in? qm?
No, in `quay.io/centos-sig-automotive/automotive-osbuild`

Reply:
a bad merge to main is exactly what you want to detect anyway, isn't it?
curl -O https://gitlab.com/CentOS/automotive/sample-images/-/raw/main/auto-image-builder.sh?ref_type=heads would work...

Reply:
Done.
Took some time as I was in some massive virus knockout